hbase-hadoop-compat/src/main/java/org/apache/hadoop/hbase/regionserver/MetricsRegionSource.java

```diff
@@ -53,6 +53,10 @@
   String COPROCESSOR_EXECUTION_STATISTICS_DESC = "Statistics for coprocessor execution times";
   String REPLICA_ID = "replicaid";
   String REPLICA_ID_DESC = "The replica ID of a region. 0 is primary, otherwise is secondary";
+  String READ_REQUEST_ON_MEMSTORE = "readRequestCountOnMemstore";
+  String READ_REQUEST_ON_MEMSTORE_DESC = "Reads happening out of memstore";
+  String MIXED_READ_REQUEST_ON_STORE = "mixedReadRequestCountOnStore";
+  String MIXED_READ_REQUEST_ON_STORE_DESC = "Reads happening out of files and memstore on store";
```

Comment:
Yeah, don't we have this accounted already? Seems redundant.

Reply:
The one above is not needed - MEMSTORE_GET_KEY  and FILE_GET_KEY  in MetricsREgionServerSource because I had removed all those additional metric.

Reply:
That constants I did not clean up.